src/umi_tools/umi_tools_extract/config.vsh.yaml

```diff
@@ -0,0 +1,197 @@
+name: umi_tools_extract
+namespace: umi_tools
+description: |
+  Flexible removal of UMI sequences from fastq reads.
+  UMIs are removed and appended to the read name. Any other barcode, for example a library barcode,
+  is left on the read. Can also filter reads by quality or against a whitelist.
+keywords: [ extract, umi-tools, umi, fastq ]
+links:
+  homepage: https://umi-tools.readthedocs.io/en/latest/
+  documentation: https://umi-tools.readthedocs.io/en/latest/reference/extract.html
+  repository: https://github.com/CGATOxford/UMI-tools
+references: 
+  doi: 10.1101/gr.209601.116
+license: MIT
+
+argument_groups:
+
+  - name: Input
+    arguments: 
+    - name: --paired
+      type: boolean_true
+      description: Paired fastq files. If option is set, two input files are expected.
+    - name: --input
+      type: file
+      required: true
+      description: Input fastq files, either one or two (paired).
+      example: sample.fastq
+    - name: --read2_in
+      type: file
+      description: Filename for read pairs.
+    - name: --bc_pattern
+      type: string
+      description: |
+        The UMI barcode pattern to use e.g. 'NNNNNN' indicates that the first 6 nucleotides 
+        of the read are from the UMI.
+    - name: --bc_pattern2
+      type: string
+      description: The UMI barcode pattern to use for read 2.
+    
+  - name: "Output"
+    arguments:  
+    - name: --read1_out
+      type: file
+      required: true
+      description: Output file for read 1.
+      direction: output
+    - name: --read2_out
+      type: file
+      description: Output file for read 2.
+      direction: output
```

Comment:
Currently:

`umi_tools extract`:
  - Inputs:
    * R1: `-I`
    * R2: `--read2-in`
  - Outputs:
    * R1: `-S`
    * R2: `--read2-out`

This component:
  - Inputs:
    * R1: `--input`
    * R2: `--read2_in`
  - Outputs:
    * R1: `--read1_out`
    * R2: `--read2_out`

I propose changing the inputs of the component to something consistent. Options:

* `--input`, `--read2_in`, `--output`, `--read2_out`
* `--input`, `--input_r2`, `--output`, `--output_r2`